lib/project_types/node/messages/messages.rb

```diff
@@ -83,52 +83,18 @@ module Messages
 
         generate: {
           help: <<~HELP,
-          Generate code in your Node project. Supports generating new billing API calls, new pages, or new webhooks.
-            Usage: {{command:%s generate [ billing | page | webhook ]}}
-          HELP
-          extended_help: <<~EXAMPLES,
-          {{bold:Examples:}}
-            {{cyan:%s generate webhook PRODUCTS_CREATE}}
-              Generate and register a new webhook that will be called every time a new product is created on your store.
-          EXAMPLES
+          {{red:The {{command:generate}} command will no longer be supported going forward, because the new Shopify libraries will provide these features instead.}}
```

Comment:
I think we might be trying to over-explain here, and it's not really working because we don't have specific information to share. In which case let's keep it short and to the point.

```suggestion
          {{red:The {{command:generate}} command is no longer supported.}}
```